features/dom.yml.dist

```diff
@@ -0,0 +1,1220 @@
+  # baseline_high_date: 2020-10-30
+  # support:
+  #   chrome: "61"
+  #   chrome_android: "61"
+  #   edge: "17"
+  #   firefox: "49"
+  #   firefox_android: "49"
+  #   safari: "10.1"
+  #   safari_ios: "10.3"
+  - api.DOMTokenList.replace
+
+  # baseline: high
+  # baseline_low_date: 2018-05-09
+  # baseline_high_date: 2020-11-09
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "60"
+  #   firefox_android: "60"
+  #   safari: "1"
+  #   safari_ios: "1"
+  - api.Document.body
+
+  # baseline: high
+  # baseline_low_date: 2018-10-02
+  # baseline_high_date: 2021-04-02
+  # support:
+  #   chrome: "61"
+  #   chrome_android: "61"
+  #   edge: "18"
+  #   firefox: "45"
+  #   firefox_android: "45"
+  #   safari: "10.1"
+  #   safari_ios: "10.3"
+  - api.Element.getAttributeNames
+
+  # baseline: high
+  # baseline_low_date: 2018-10-02
+  # baseline_high_date: 2021-04-02
+  # support:
+  #   chrome: "67"
+  #   chrome_android: "67"
+  #   edge: "18"
+  #   firefox: "61"
+  #   firefox_android: "61"
+  #   safari: "12"
+  #   safari_ios: "12"
+  - api.DOMTokenList.replace.boolean_value
+
+  # baseline: high
+  # baseline_low_date: 2018-10-23
+  # baseline_high_date: 2021-04-23
+  # support:
+  #   chrome: "69"
+  #   chrome_android: "69"
+  #   edge: "18"
+  #   firefox: "63"
+  #   firefox_android: "63"
+  #   safari: "12"
+  #   safari_ios: "12"
+  - api.Element.toggleAttribute
+
+  # baseline: high
+  # baseline_low_date: 2020-01-15
+  # baseline_high_date: 2022-07-15
+  # support:
+  #   chrome: "51"
+  #   chrome_android: "51"
+  #   edge: "79"
+  #   firefox: "36"
+  #   firefox_android: "36"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - api.NodeList.@@iterator
+
+  # baseline: high
+  # baseline_low_date: 2020-10-20
+  # baseline_high_date: 2023-04-20
+  # support:
+  #   chrome: "86"
+  #   chrome_android: "86"
+  #   edge: "86"
+  #   firefox: "78"
+  #   firefox_android: "79"
+  #   safari: "14"
+  #   safari_ios: "14"
+  - api.Document.replaceChildren
+  - api.DocumentFragment.replaceChildren
+  - api.Element.replaceChildren
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "62"
+  #   chrome_android: "62"
+  #   edge: "18"
+  #   firefox: "56"
+  #   firefox_android: "56"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.Document.visibilitychange_event
+
+  # baseline: high
+  # baseline_low_date: 2022-03-08
+  # baseline_high_date: 2024-09-08
+  # support:
+  #   chrome: "1"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "98"
+  #   firefox_android: "98"
+  #   safari: "1.3"
+  #   safari_ios: "1"
+  - api.HTMLElement.outerText
+
+  # baseline: low
+  # baseline_low_date: 2024-09-16
+  # support:
+  #   chrome: "125"
+  #   chrome_android: "125"
+  #   edge: "125"
+  #   firefox: "128"
+  #   firefox_android: "128"
+  #   safari: "18"
+  #   safari_ios: "18"
+  - api.Element.getHTML
```

Comment:
This could be moved out maybe? Possibly belongs to Declarative Shadow DOM?  https://chromestatus.com/feature/5102952270528512

Reply:
Fixed with d3228d5. (I also took out the keys after these two, which turned out to be for custom elements.)